components/common_macro/gazebo_object_controller_macro.xacro

```diff
@@ -29,7 +29,7 @@
     
                 <!-- Frame IDs -->
                 <odometryFrame>${odom_frame}</odometryFrame>
```

Comment:
I think odometryFrame needs to be changed as well..

Reply:
Yes Pradheep you are right. I have changed it.